src/sgame/sg_cmds.cpp

```diff
@@ -2345,6 +2345,9 @@ static bool Cmd_Class_internal( gentity_t *ent, const char *s, bool report )
 
 			num = trap_EntitiesInBox( mins, maxs, entityList, MAX_GENTITIES );
 
+			float aliensInRange = 0;
+			float humansInRange = 0;
```

Comment:
These should both be ints since you can't have partial humans...

Reply:
ok 😅